### PR TITLE
V2021.8

### PR DESCRIPTION
**Pull request by JuliaRS**

### CODE REVIEW

File: src_c/IMB_output.c [hunk 1]
```diff
@@ -342,7 +342,11 @@ Input variables:
                 break;
 
             case SAMPLE_FAILED_INT_OVERFLOW:
-                sprintf(aux_string + offset, " int-overflow.; The production rank*size caused int overflow for given sample");
+#ifdef MPI1
+                sprintf(aux_string + offset, " int-overflow; The production rank*size caused int overflow for given sample; use flag \"-data_type double\"");
```
Comment (Sergei-Lebedev) [inline src_c/IMB_output.c:346]: seems like there is aux_string overflow in overflow message. aux_string is 104 bytes, but the string " int-overflow; The production rank*size caused int overflow for given sample; use flag \"-data_type double\"" is 106 bytes

Comment (Sergei-Lebedev) [reply src_c/IMB_output.c:346]: just gcc compilation fails with two errors
```
../src_c/IMB_chk_diff.c:1065:13: error: this ‘if’ clause does not guard... [-Werror=misleading-indentation]
 1065 |             if (ranks[i + 1] == rank && pos2 - pos1 + 1 == (lengths[i] + (lengths[i + 1])))
      |             ^~
../src_c/IMB_chk_diff.c:1066:41: note: ...this statement, but the latter is misleadingly indented as if it were guarded by the ‘if’
 1066 |                 c_info->reccnt[rank]++; i++;
      |                                         ^
/tmp/ompi/install/bin/mpicc -g -O0 -Wall -Werror -DCHECK=1 -Ihelpers -I../src_c -DMPI1 -I. -DMPI1 -c -o MPI1/CPU/IMB_warm_up.o ../src_c/IMB_warm_up.c
/tmp/ompi/install/bin/mpicxx -DCHECK=1 -Ihelpers -I../src_c -DMPI1 -I. -g -O0 -Wall -Wextra -Werror -DMPI1 -c -o MPI1/CPU/MPI1_suite.o MPI1/MPI1_suite.cpp
/tmp/ompi/install/bin/mpicxx -DCHECK=1 -Ihelpers -I../src_c -DMPI1 -I. -g -O0 -Wall -Wextra -Werror -DMPI1 -c -o MPI1/CPU/MPI1_benchmark.o MPI1/MPI1_benchmark.cpp
../src_c/IMB_output.c: In function ‘IMB_display_times’:
../src_c/IMB_output.c:346:152: error: ‘ int-overflow; The productio...’ directive writing 106 bytes into a region of size 104 [-Werror=format-overflow=]
  346 |                 sprintf(aux_string + offset, " int-overflow; The production rank*size caused int overflow for given sample; use flag \"-data_type double\"");
      |                                               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~
../src_c/IMB_output.c:346:17: note: ‘sprintf’ output 107 bytes into a destination of size 104
  346 |                 sprintf(aux_string + offset, " int-overflow; The production rank*size caused int overflow for given sample; use flag \"-data_type double\"");
      |                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment (JuliaRS) [reply src_c/IMB_output.c:346]: @Sergei-Lebedev please try to use branch from https://github.com/intel/mpi-benchmarks/pull/54 and let me is it works for you or not.

Comment (Sergei-Lebedev) [reply src_c/IMB_output.c:346]: @JuliaRS, thanks for the quick fix, it helped

**Pull request merged**